### PR TITLE
Claudy improvements

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

**Comment by CPatchane** [comment]
[(5) [NUA-7] ETU je peux contacter le support directement depuis le nuage](https://trello.com/c/aGWhw9XW/541-5-nua-7-etu-je-peux-contacter-le-support-directement-depuis-le-nuage)

File: src/services/ClaudyActionComponents/Support.jsx [hunk 1]
```diff
@@ -77,12 +82,14 @@ export class Support extends Component {
                 onChange={(e) => { this.setState({message: e.target.value}) }}
               />
             </label>
-            {!isSent && !isSending && !error &&
+            {(
+              (!isSent && !isSending && !error) ||
+              (isSent && !isSending && !error && message)) &&
```
Comment (gregorylegarec) [inline src/services/ClaudyActionComponents/Support.jsx]: Shouldn't we be using some kind of selectors here ? The code readibility is not optimal imho.

**Comment by gregorylegarec** [comment]
Thanks @CPatchane 🍨

File: src/services/ClaudyActionComponents/Support.jsx (hunk 1 above)
Comment (CPatchane) [inline src/services/ClaudyActionComponents/Support.jsx]: Agree, I'll think about it if I'll need to update that so. 👍

**Pull request merged**